src/BaseTemplates/WebAPI/project.json

```diff
@@ -36,7 +36,10 @@ $if$ ($context$ == WebCore) $else$
   },
 $endif$
   "tools": {
-    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*"
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
+      "version": "__AspNetCorePlatformVersion__",
+      "imports": "portable-net45+wp80+win8+wpa81+dnxcore50"
```

Comment:
Aren't these imports for `Newtonsoft.Json`? If so `portable-net45+win8+dnxcore50` will suffice